Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -195,6 +195,14 @@
   description: These protect you from slips while looking like normal sneakers.
   price: 2
 
+- type: uplinkListing
+  id: UplinkClothingEyesXrayMesons
+  category: Armor
+  itemId: ClothingEyesGlassesMesonXRay
+  listingName: x-ray messon goggles
+  description: These let you see everything through walls, and give you flash resistance.
+  price: 5
```

Comment:
If we end up keeping xray goggles then this either needs to be much more expensive, the range needs a nerf, or thermals need to be dropped entirely to balance it.

Reply:
I asked on Discord how expensive it should be and this was the most common answer, I can make it more expensive if you want though.